src/components/_tabs.scss

```
@@ -0,0 +1,79 @@
+// Tabs
+//
+// Tabs is a simple tabs component which allows you to show one piece of content at a time
+//
+// JavaScript is required to actually add/remove the href and classes and position/animate the
+// line on the active item. In order to animate the line, scale + translate should be used to
+// leverage hardware acceleration.
+//
+// Browser support: IE11+, Safari6.1+, Firefox22+, Chrome21+
+//
+// Markup:
+// <nav class="sb-tabs sb-text--smallcaps">
+//   <div class="sb-tabs__inner">
+//     <a href="#chart-type" class="sb-tabs__tab--active">Chart type</a>
```

```suggestion
//     <a href="#chart-type" class="sb-tab-link sb-tab-link--active">Chart type</a>
```
giving the explicit classnames avoid the nesting in styling.

---

I thought about doing this but I thought it felt a bit excessive to have classes assigned even for the default links. That said, it if more explicit so probably worth it - it seems this is the way it is done everywhere anyway. :)

---

I've updated this now btw - I didn't accept the suggestion cause there were also style changes involved.